controller/daemon.go

```diff
@@ -38,7 +41,13 @@ func NewDaemon(conf *config.Config) (*Daemon, error) {
 
 	s := state.NewInMemoryStore(conf)
 
-	tm, err := NewTasksManager(conf, s)
+	logger.Info("initializing Consul client and testing connection")
```

Comment:
Can we add a debug log to log which controller was initialized?

Reply:
same for other controllers

Reply:
There is a log that lets the user know which controller was initialized ~5 lines above. Do you think that would be sufficient?
`logger.Info("setting up controller", "type", "daemon")`

Reply:
oh yep, all good!